doc/classes/Basis.xml

```diff
@@ -281,34 +301,48 @@
 			<param index="0" name="to" type="Basis" />
 			<param index="1" name="weight" type="float" />
 			<description>
-				Assuming that the matrix is a proper rotation matrix, slerp performs a spherical-linear interpolation with another rotation matrix.
+				Performs a spherical-linear interpolation with the [param to] basis, given a [param weight]. Both this basis and [param to] should represent a rotation.
+				[b]Example:[/b] Smoothly rotate a [Node3D] to the target basis over time, with a [Tween].
+				[codeblock]
+				var start_basis = Basis.IDENTITY
+				var target_basis = Basis.IDENTITY.rotated(Vector3.UP, TAU / 2)
+
+				func _ready():
+				    create_tween().tween_method(interpolate, 0.0, 1.0, 5.0).set_trans(Tween.TRANS_EXPO)
+
+				func interpolate(weight):
+				    basis = start_basis.slerp(target_basis, weight)
+				[/codeblock]
 			</description>
 		</method>
 		<method name="tdotx" qualifiers="const">
 			<return type="float" />
 			<param index="0" name="with" type="Vector3" />
 			<description>
-				Transposed dot product with the X axis of the matrix.
+				Returns the transposed dot product between [param with] and the [member x] axis (see [method transposed]).
+				This is equivalent to [code]basis.x.x * vector.x + basis.y.x * vector.y + basis.z.x * vector.z[/code].
```

Comment:
This is the opposite of what it should be. I think you got tripped up by the C++ code using rows. `rows[1][0]` in C++ is `basis.x.y` in GDScript.

```suggestion
				This is equivalent to [code]basis.x.dot(vector)[/code].
```

Honestly, we should consider deprecating this method. It doesn't seem very useful. `basis.tdotx(vector)` is the same amount of characters as `basis.x.dot(vector)`, and the engine itself never uses this method outside of internal calculations in Basis itself.

Reply:
I'd be almost all up for it. I don't see the point of this method for most users, although it may run faster.